src/khoj/routers/api.py

```diff
@@ -149,7 +149,7 @@ async def execute_search(
     encoded_asymmetric_query = None
     if t != SearchType.Image:
         with timer("Encoding query took", logger=logger):
-            search_model = await sync_to_async(get_user_search_model_or_default)(user)
+            search_model = await sync_to_async(get_default_search_model)()
```

Comment:
should this be updated to use `get_user_default_search_model` as well to encode user query for now?